spec/vast_trackers.spec.js

```diff
@@ -0,0 +1,147 @@
+import { VASTClient } from '../src/vast_client';
+import { VASTParser } from '../src/parser/vast_parser';
+import { VASTTracker } from '../src/vast_tracker';
+import { inlineTrackers } from '../spec/samples/inline_trackers';
+
+const vastParser = new VASTParser();
+const vastClient = new VASTClient();
+
+describe('VASTTracker', function() {
```

Comment:
Ideally for VASTTracker, we should test with already parsed VASTs, see https://github.com/dailymotion/vast-client-js/issues/264.

Reply:
Done !